model_compression_toolkit/core/pytorch/reader/reader.py

```diff
@@ -85,7 +85,7 @@ def fx_graph_module_generation(pytorch_model: torch.nn.Module,
     """
     set_model(pytorch_model)
     symbolic_traced = symbolic_trace(pytorch_model)
-    inputs = representative_data_gen()
+    inputs = representative_data_gen().__next__()
```

Comment:
I think that `next(representative_data_gen())` is the more convenient form of using a generator